integration-tests/src/setup.rs

```diff
@@ -392,6 +408,44 @@ impl ExtBuilder {
                 dot(100f64),
             )
             .unwrap();
+
+            //initialize for clv as mock sibling
+            Assets::force_create(
+                Origin::root(),
+                CLV,
+                MultiAddress::Id(AccountId::from(ALICE)),
+                true,
+                1,
+            )
+            .unwrap();
+            Assets::force_set_metadata(
+                Origin::root(),
+                CLV,
+                b"CLV".to_vec(),
+                b"CLV".to_vec(),
+                CLV_DECIMAL,
+                false,
+            )
+            .unwrap();
+            let clv_asset_location = MultiLocation::new(
+                1,
+                X2(
+                    Parachain(2002),
+                    //since we use para to mock clv,just use para location here
+                    GeneralKey(WeakBoundedVec::<u8, ConstU32<32>>::force_from(
+                        b"PARA".to_vec(),
+                        None,
+                    )),
```

Comment:
Looks like if using `b"PARA"` here, then this line will take effects:
https://github.com/parallel-finance/parallel/blob/1a93d800a816ae307ea0c6e3a32e2f30d15f6345/runtime/parallel/src/lib.rs#L492-L497

`AssetRegistry` will not take any effect here.

Reply:
yes it's only used as fallback